aspnet/fundamentals/hosting.rst

```diff
@@ -0,0 +1,226 @@
+:version: 1.0.0
+
+Hosting
+=======
+
+By `Steve Smith`_
+
+To run an ASP.NET Core app, you need to configure and launch a host using `WebHostBuilder <https://docs.asp.net/projects/api/en/latest/autoapi/Microsoft/AspNetCore/Hosting/WebHostBuilder/index.html?highlight=webhostbuilder>`_.
```

Comment:
Don't think you need the highlight query string here. All the highlights makes it rather distracting.